packages/jest-circus/src/utils.js

```diff
@@ -0,0 +1,225 @@
+/**
```

Comment:
I really hate utils methods. Can we split up this file into many small modules or colocate the functions with where they are needed? I really want Jest to get rid of the word "util" in packages and file names entirely.